xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/uicomponents/search/searchSuggest.js

```diff
@@ -118,7 +118,8 @@ var XWiki = (function (XWiki) {
      * Callback triggered when the suggest element is collapsed, because of a focusout event for example.
      */
     onSuggestCollapsed: function(event) {
-      this.suggest.clearSuggestions();
+      /* We match the collapse of the input field timing to close the suggest panel. */
```

Comment:
Where is located the duration of the input field timing? Not clear to me when reading the changes where I could find this `300` value.

Reply:
Updated the comment to be more explicit :+1: